doc/src/manual/noteworthy-differences.md

```diff
@@ -211,7 +211,7 @@ For users coming to Julia from R, these are some noteworthy differences:
     by default. To get optimal performance when looping over arrays, the order of the loops should
     be reversed in Julia relative to NumPy (see relevant section of [Performance Tips](@ref man-performance-tips)).
   * Julia's updating operators (e.g. `+=`, `-=`, ...) are *not in-place* whereas NumPy's are. This
-    means `A = ones(4); B = A; B += 3` doesn't change values in `A`, it rather rebinds the name `B`
+    means `A = fill(1, 4); B = A; B .+= 3` doesn't change values in `A`, it rather rebinds the name `B`
```

Comment:
Good catch re. `+=`!

Reply:
Ah, I hadn't read the rest of the sentence :). `+=` is indeed correct, rather than `.+=`?

Reply:
Right, the reason I changed was that vector + scalar is deprecated, so perhaps we should just rewrite the example completely.

Reply:
Perfect! :)